dev/versions.md

```diff
@@ -151,6 +151,7 @@ Note: overriding horizontal-pod-autoscaler-sync-period on EKS is currently not s
 
 1. Find the latest release on [Dockerhub](https://hub.docker.com/r/fluent/fluentd-kubernetes-daemonset/)
 1. Update the base image version in `images/fluentd/Dockerfile`
+1. Update fluent-gem versions to latest
```

Comment:
Can you mention the file path (`images/fluentd/Dockerfile` and link to the URL to find the latest version?)